### ISSUE TITLE
Changing the video shows the new thumbnail and the old video embed

### ISSUE THREAD

**Issue by ronnyroeller** [GoogleWebComponents/google-youtube#46]
I added to the (stripped down) demo a button which allows changing the video:

```javascript
<link rel="import" href="../../polymer/polymer.html">
<link rel="import" href="../google-youtube.html">

<dom-module id="demo-element">
  <template id="page-template">
    <google-youtube id="custom"
    				video-id="yRbOSdAe_JU"
                    thumbnail="//www.polymer-project.org/images/logos/p-logo.svg">
    </google-youtube>
    
    <button on-tap='changeVideo'>Change video</button>
  </template>
</dom-module>
<script>
  Polymer({
    is: 'demo-element',
    changeVideo: function(ev) {
    	this.$.custom.videoid = '0GxteaIaj2Q';
    	this.$.custom.thumbnail = 'http://i.ytimg.com/vi/0GxteaIaj2Q/hqdefault.jpg';
    }
  });
</script>
```

Initially, the user sees the first thumbnail:
![youtube-bug-1](https://cloud.githubusercontent.com/assets/9339055/10727666/dc94d75a-7bdb-11e5-93ce-891e916d32a2.png)

Clicking on the thumbnail starts the correct video:
![youtube-bug-2](https://cloud.githubusercontent.com/assets/9339055/10727665/dc94376e-7bdb-11e5-9423-e921141b7a9c.png)

After clicking the button to change the video, the new thumbnail is shown but the old video embed is still visible:
![youtube-bug-3](https://cloud.githubusercontent.com/assets/9339055/10727664/dc92d04a-7bdb-11e5-9b12-6c2456a63708.png)

I would expect to see only the new thumbnail. I would expect that the video embed is no longer in the DOM.

**Comment by jeffposnick** [comment]
There's a bug in the demo, with `videoid` being used incorrectly when `videoId` should be used: https://github.com/GoogleWebComponents/google-youtube/pull/47

I see that you recently edited this issue—in the initial version, were you using `videoid` or `videoId`? Does using `videoId` fix things?

**Comment by ronnyroeller** [comment]
@jeffposnick, you are right: It was initially wrong but I corrected the snippet and the screenshots in the ticket. Problem is still there doe.

**Comment by ronnyroeller** [comment]
I created a Pull Request that fixes this issue: https://github.com/GoogleWebComponents/google-youtube/pull/48

**Comment by kovacicmilan** [comment]
Is there any update on this?
